ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -29,26 +29,41 @@ chain:
       echo "Waiting for cluster to become available"
       oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${KUBEVIRT_CLUSTER_NAME}
       echo "Cluster became available, creating kubeconfig"
-      bin/hypershift create kubeconfig --name=${KUBEVIRT_CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
+      bin/hypershift create kubeconfig --name=${KUBEVIRT_CLUSTER_NAME} >${SHARED_DIR}/guest_kubeconfig
 
-      echo "Waiting for nested cluster's node count to reach the desired replicas count in the NodePool"
+      echo "Waiting for guest cluster's node count to reach the desired replicas count in the NodePool"
       until \
           [[ $(oc get nodepool ${KUBEVIRT_CLUSTER_NAME} -n clusters -o jsonpath='{.spec.replicas}') \
-            == $(oc --kubeconfig=${SHARED_DIR}/nested_kubeconfig get nodes --no-headers | wc -l) ]]; do
-              echo "$(date --rfc-3339=seconds) Nested cluster's node count is not equal to the desired replicas in the NodePool. Retrying in 30 seconds."
+            == $(oc --kubeconfig=${SHARED_DIR}/guest_kubeconfig get nodes --no-headers | wc -l) ]]; do
+              echo "$(date --rfc-3339=seconds) guest cluster's node count is not equal to the desired replicas in the NodePool. Retrying in 30 seconds."
               oc get vmi -n ${CLUSTER_NAMESPACE}
               sleep 30s
       done
 
       echo "Waiting for clusteroperators to be ready"
-      export KUBECONFIG=${SHARED_DIR}/nested_kubeconfig
+
+      if ! timeout 15m bash -c 'until \
+        oc --kubeconfig ${SHARED_DIR}/guest_kubeconfig wait clusterversion/version --for='condition=Available=True' > /dev/null;  do
+          echo "$(date --rfc-3339=seconds) Cluster Operators not yet ready"
+          oc --kubeconfig ${SHARED_DIR}/guest_kubeconfig get clusteroperators 2>/dev/null || true
+          sleep 1s
+      done'; then
+        VMs=$(oc get vm -o name)
+        for vm in ${VMs[@]}
+        do
+           oc patch ${vm} --type=json -p '[{"op": "replace", "path": "/spec/runStrategy", "value": "Halted"}]'
+           sleep 1s
+           oc patch ${vm} --type=json -p '[{"op": "replace", "path": "/spec/runStrategy", "value": "Always"}]'
+        done
+      fi
```

Comment:
given what we know so far, there's a chance this does in fact reduce the flakiness involved with console operator coming online. Obviously this is treating the symptom instead of the cause though.